pkg/security/secl/rules/ruleset.go

```diff
@@ -104,17 +122,33 @@ func (rd *RuleDefinition) GetTag(tagKey string) (string, bool) {
 	return "", false
 }
 
+func applyOverride(rd1, rd2 *RuleDefinition) {
+	// for backward compatibility, by default only the expression is copied if no options
+	if len(rd2.OverrideOptions.Fields) == 0 {
+		rd1.Expression = rd2.Expression
+	} else if slices.Contains(rd2.OverrideOptions.Fields, OverrideAllFields) {
+		*rd1 = *rd2
+	} else {
+		if slices.Contains(rd2.OverrideOptions.Fields, OverrideExpressionField) {
+			rd1.Expression = rd2.Expression
+		}
+		if slices.Contains(rd2.OverrideOptions.Fields, OverrideActionFields) {
+			rd1.Actions = rd2.Actions
+		}
+		if slices.Contains(rd2.OverrideOptions.Fields, OverrideEveryField) {
+			rd1.Every = rd2.Every
```

Comment:
Is it something that is already supported by the agent or do you need some additional logic to handle the override?

Reply:
Nothing specific here, see my previous comment